api/src/main/java/org/openmrs/module/eptsreports/reporting/library/dimensions/EptsCommonDimension.java

```diff
@@ -77,7 +77,7 @@ public CohortDefinitionDimension age(AgeDimensionCohortInterface ageDimensionCoh
     dim.addCohortDefinition(
         "5-9", ageDimensionCohort.createXtoYAgeCohort("patients with age between 5 and 9", 5, 9));
     dim.addCohortDefinition(
-        "<15", ageDimensionCohort.createXtoYAgeCohort("patients with age below 15", 0, 15));
+        "<15", ageDimensionCohort.createXtoYAgeCohort("patients with age below 15", 0, 14));
```

Comment:
please revert this, i have put it MEPTS-151-user-stories-updates since i need it there for testing. sorry